### ISSUE TITLE
"Send request" button is not accessible using keyboard TAB keys.:A11y_VS Code extension for API Center_OpenAPIDocumentation_CodeExecute_Keyboard

### ISSUE THREAD

**Issue by PoojaNamde** [microsoft/vscode-azureapicenter#138]
"[Check out Accessibility Insights](https://nam06.safelinks.protection.outlook.com/?url=https://accessibilityinsights.io/&data=02%7c01%7cv-stfe%40microsoft.com%7cb67b2c4b646d4f9561a208d6f4b5c39b%7c72f988bf86f141af91ab2d7cd011db47%7c1%7c0%7c636965458850501301&sdata=mxhokIKNMb22llsjXHLgU3XZibj1Qfx37rpY4PU2sfE%3D&reserved=0)! - Identify accessibility bugs before check-in and make bug fixing faster and easier."

**Please do not close this bug. This bug should only be closed by Trusted Tester after verification.**

### GitHub Tags
#A11yTCS; #A11yMAS; #A11ySev2; #WCAG2.4.7; #GH_VSCodeextensionforAPICenter_Linux_Apr2024; #DesktopApp; #VS Code Extension for API Center; #Keyboard; #Linux; #FTP;

### Environment Details:
Application Name: VS Code Extension for API Center
OS: Linux- Ubuntu 22.04.4

### Repro Steps:

1. Create a API center in azure portal.
2. Open Visual studio Code.
3. Tab till API center extension in the left side panel and press enter.
4. Tab till definition under the api name which you have created and press enter.
5. The name which you have provided to the definition will appear for ex-"openapi4", right click on it.
6. Go to "Generate HTTP file" using the down arrow and press enter.
7. A HTTP file will appear. Try to navigate to "Send request" button and observe that "Send request" button is not accessible using keyboard.

### Actual Result:
"Send request" button is not accessible using keyboard TAB keys.

### Expected Result:
"Send request" button should be accessible using keyboard TAB keys.

### User Impact:
Keyboard only users will not be able to access the sendbutton and will not be able to perform the further action.

### Attachment:
![image](https://github.com/microsoft/vscode-azureapicenter/assets/90901519/6cbe4cc4-f6e9-46c7-bf3c-500c82064751)

**Comment by formulahendry** [comment]
External issue. This UI control is from REST Client extension: https://github.com/Huachao/vscode-restclient

**Comment by AnKushSingh05** [comment]
> External issue. This UI control is from REST Client extension: https://github.com/Huachao/vscode-restclient

Can you please provide the service tree details so we can route this to external team?.

**Comment by Yash14j** [comment]
@formulahendry As per the C+AI accessibility testing external bug process primary team need to file the bug to VSCode team & share us the external bug ID.

**Comment by formulahendry** [comment]
This issue is caused by UI of REST Client extension, not related to UI of API Center extension.

**Comment by AnKushSingh05** [comment]
GitHub Tags:
#eDAD:3P:Bug223550;#Thirdparty;#External;#External:RestClient;

IRT Bug Logged: https://dev.azure.com/CSS-IRT/CSS%20IRT/_workitems/edit/223550

**Comment by AnKushSingh05** [comment]
#Closed; 
Closing this bug as this is not in the scope of API center
